VndbSharp/VndbUtils.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using VndbSharp.Structs.Models.Dumps;
+
+namespace VndbSharp
+{
+    public class VndbUtils
+    {
+        protected DateTime LastTime = DateTime.Now;
+        protected static Boolean FirstRun = true;
```

Comment:
I think I fixed it with this commit:
https://github.com/micah686/VndbSharp/commit/6bb9a3a53597c57654f8d7c9dd7615294131168f